uaclient/security.py

```diff
@@ -572,6 +574,23 @@ def fix_security_issue_id(
                     issue_id=issue_id
                 )
             raise exceptions.UserFacingError(msg)
+
+        # Check livepatch status for CVE in fixes
+        status = subprocess.run(
+            ["canonical-livepatch", "status", "--verbose", "--format=json"],
+            capture_output=True,
+            encoding="utf-8",
+        ).stdout
+        if status:
+            parsed_patch = json.loads(status)["Status"][0]["Livepatch"]
+            if cve.id.lower() in parsed_patch.get("Fixes", ""):
```

Comment:
Thanks! Was having difficulty finding documentation on the output of the livepatch command itself as my personal Linux box isn't missing any patches

Reply:
Yep the documentation is a bit lacking there. For future reference, I got that example by creating a 16.04/xenial vm with [multipass](https://multipass.run/) (`multipass launch xenial -n dev-x && multipass shell dev-x`) and enabling livepatch on it. Since the xenial image has an old kernel, livepatch immediately patched it.